types/core/config.d.ts

```diff
@@ -8,16 +8,16 @@ declare const CONFIG: {
    */
   Actor: {
     collection: Actors;
-    entityClass: typeof Actor;
-    sheetClasses: typeof ActorSheet;
-    typeLabels: any;
+    entityClass: ConstructorOf<Actor>;
+    sheetClasses: Record<string, unknown>;
```

Comment:
Wouldn't the `unknown` also be a `ConstructorOf<...>`?

Reply:
The sheetclasses parameter is a nested object.
![image](https://user-images.githubusercontent.com/182554/106957968-4fb63680-66fe-11eb-9e1e-95aa8510b938.png)

Reply:
Is there some required structure to the nested stuff? Should we maybe add a TODO to improve the type once we know more about the structure?

Reply:
I'll add a TODO. Actually typing the structure there is a bit beyond the scope on this PR.